src/message.ts

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+import get from 'lodash.get';
+import { CumulusMessage } from '@cumulus/types/message';
+import { cumulusMessageWithGranulesInPayload } from './types';
+
+const GRANULE_LOG_LIMIT = 500;
+
+/**
+ * Get granules from execution message.
+ *   Uses the order of precedence as defined by the cumulus/common/message
+ *   description.
+ *
+ * @param {Object} message - a Cumulus message
+ * @param {integer} granuleLimit - number of granules to limit the log to
+ * including, to avoid environment variable truncation
+ * @returns {Array<Object>} - An array of granule ids
+ */
+export const getMessageGranules = (
+  message: cumulusMessageWithGranulesInPayload,
+  granuleLimit: number = GRANULE_LOG_LIMIT
+) => {
```

Comment:
Please add an explicit return type.

Reply:
420deba - good catch, this highlighted a bad typing assumption.